firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -162,7 +162,53 @@ public Task<FetchResponse> fetch(long minimumFetchIntervalInSeconds) {
             executor,
             (cachedFetchConfigsTask) ->
                 fetchIfCacheExpiredAndNotThrottled(
-                    cachedFetchConfigsTask, minimumFetchIntervalInSeconds));
+                    cachedFetchConfigsTask,
+                    minimumFetchIntervalInSeconds, /*excludeEtagHeader */
```

Comment:
wonder if we can get the formatter to leave this on the right line... no need to worry about it right now